app/utils/transformer/DepartureDataTransformer.scala

```diff
@@ -43,7 +50,9 @@ class DepartureDataTransformer @Inject() (
       transportEquipmentTransformer.transform andThen
       containerIdTransformer.transform andThen
       sealTransformer.transform andThen
-      limitDateTransformer.transform
+      limitDateTransformer.transform andThen
+      modeOfTransportAtTheBorderTransformer.transform
+    addBorderModeOfTransportYesNoTransformer.transform
```

Comment:
Are we perhaps missing a `andThen` here?

Reply:
Yeah, just amended that

Reply:
@joecefnhedog can the clean-up logic on the AddBorderModeOfTransportYesNoPage be then updated?

Reply:
Good spot, I reckon we remove this functionality to ensure we have captured all the cases where 15 data may be getting removed.

Reply:
Once all the pages that are affected by the clean-up logic have started using the transform implementations then I believe the logic can be updated.